common/src/thrift/file_system_master.thrift

```diff
@@ -23,6 +23,7 @@ struct FileInfo {
   18: string userName
   19: string groupName
   20: i32 permission
+  21: string persistenceState
```

Comment:
It could be worth using a thrift enum so that it's easier to handle backcompat for this field